src/main/package.json

```diff
@@ -70,7 +71,7 @@
   "scripts": {
     "start": "node main.js 2>&1 | pino-pretty",
     "test": "jest",
-    "dev": "node main.js 2>&1 | pino-pretty"
+    "dev": "nodemon  main.js 2>&1 | pino-pretty"
```

Comment:
Why suddenly do we need to change the way we run the server??😰😰

Reply:
When you save a change, it restart it.